projects/aca-content/src/lib/components/details/details.component.html

```diff
@@ -8,18 +8,21 @@
     <div class="acs-details-container">
       <div class="acs-details-title">
         <div class="acs-details-breadcrumb" role="heading" aria-level="2" *ngIf="node">
-          <span class="acs-details-breadcrumb-library"> {{ node.name }} </span>
-          -
-          <span class="acs-details-breadcrumb-item">{{ 'APP.INFO_DRAWER.TITLE' | translate }}</span>
+          <span class="acs-details-breadcrumb-library">
+            <img class="acs-details-breadcrumb-icon" alt="{{ 'APP.INFO_DRAWER.ICON' | translate }}" src="{{ getNodeIcon(node) }}">       
```

Comment:
can we avoid calling function in html (getNodeIcon)?

Reply:
Done